live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/model/ContentMetadata.java

```diff
@@ -41,4 +41,5 @@ public class ContentMetadata
     private final String sourceMimeType;
     private final Long size;
     private final String name;
+    private final String digestIdentifier;
```

Comment:
Let's remove this change for the time being too. Also the test changes caused by this.